dev/com.ibm.ws.container.service/src/com/ibm/ws/container/service/annocache/AnnotationsBetaHelper.java

```diff
@@ -33,9 +77,11 @@
      * product edition is set to "EARLY_ACCESS".  If available, the product
      * "com.ibm.websphere.appserver" is tested.  If that product is not
      * available, the product "io.openliberty" is tested instead.
+     *
+     * Post beta, the product edition test is no longer used.
      */
-    public static final boolean IS_LIBERTY_BETA_PRODUCT =
-        setIsLibertyBetaProduct();
+    public static final boolean IS_LIBERTY_BETA_PRODUCT = true;
+    //     setIsLibertyBetaProduct();
```

Comment:
if this is no longer used, can we remove it?

Reply:
Yes.  See the reply to the second following comment.